scripts/ensure_user_name.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Please rename to `ensure-user-name.sh`, we usually use dashes instead of underscores in shell script file names.

Please add some comments to the script that this is needed in case the container user does not have a user name such as in OpenStack etc etc.

Please `chmod u+rxw,og+rx` to clean the Travis CI.